pkg/analysis/value.go

```diff
@@ -499,3 +504,7 @@ func NodeToValue(node ast.Node, resolver Resolver) (res *Value) {
 		return defaultToValue(node)
 	}
 }
+
+func NodeToValue(node ast.Node, resolver Resolver) (res *Value) {
+	return NodeToValueImpl(node, resolver, 0)
+}
```

Comment:
I'm too new at go to know whether there's a less awkward way of emulating a 'default value' - suggestions welcome

Reply:
I'd switch `NodeToValueImpl` => `nodeToValue` and keep it private to the package